      google_utils.py bug fix #355 
   def attempt_download(weights):      # Attempt to download pretrained weights if not found locally     weights = weights.strip()     weights = weights.strip().replace("'", '')      msg = weights + ' missing, try downloading from https://drive.google.com/drive/folders/1Drs_Aiu7xx6S-ix95f9kNsA6ueKRpN2J'       r = 1     r = 1  # return      if len(weights) > 0 and not os.path.isfile(weights):          d = {'yolov3-spp.pt': '1mM67oNw4fZoIOL1c8M3hHmj66d8e-ni_',  # yolov3-spp.yaml               'yolov5s.pt': '1R5T6rIyy3lLwgFXNms8whc-387H0tMQO',  # yolov5s.yaml 
